kernel/data-pipeline/api/src/main/java/org/apache/shardingsphere/data/pipeline/api/check/consistency/DataConsistencyCheckResult.java

```diff
@@ -33,12 +31,33 @@ public final class DataConsistencyCheckResult {
     
     private final DataConsistencyContentCheckResult contentCheckResult;
     
+    private final DataConsistencyCheckIgnoredType checkIgnoredType;
+    
+    private final boolean ignored;
+    
+    public DataConsistencyCheckResult(final DataConsistencyCountCheckResult countCheckResult, final DataConsistencyContentCheckResult contentCheckResult) {
+        this.countCheckResult = countCheckResult;
+        this.contentCheckResult = contentCheckResult;
+        checkIgnoredType = null;
+        ignored = false;
+    }
+    
+    public DataConsistencyCheckResult(final DataConsistencyCheckIgnoredType checkIgnoredType) {
+        this.checkIgnoredType = checkIgnoredType;
+        ignored = true;
+        countCheckResult = null;
+        contentCheckResult = null;
+    }
+    
     /**
      * Is count and content matched.
      *
      * @return matched or not
      */
     public boolean isMatched() {
+        if (ignored || null == countCheckResult || null == contentCheckResult) {
```

Comment:
Looks it's not necessary to add ` || null == countCheckResult || null == contentCheckResult`, they're required when ignored is false

Reply:
Ok, add it just because idea warning, removed